.github/ISSUE_TEMPLATE/update-opa.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Update OpenPolicyAgent (OPA)
+about: >-
+  This template contains instructions specific to updating this product and/or
+  container image(s).
+title: >-
+  chore(opa): Update container images ahead of Stackable Release XX.(X)X
+labels: []
+projects: ['stackabletech/10']
+assignees: ''
+---
+
+Part of #xxx.
+
+> [!TIP]
+> Please add the `scheduled-for/20XX-XX` label.
+
+```[tasklist]
+### Update tasks
+- [ ] Update `versions.py` to reflect the agreed upon versions in the spreadsheet (including the removal of old versions).
+- [ ] Upload new version (see `opa/upload_new_opa_version.sh`).
+- [ ] Update other dependencies if applicable (eg: opa_bundle_builder, etc).
+- [ ] Check other operators (getting_started / kuttl) for usage of the versions. Add the PR(s) to the list below.
+- [ ] Update the version in demos. Add the PR(s) to the list below.
+```
+
+```[tasklist]
+### Related Pull Requests
+- [ ] _Link to the docker-images PR (product update)_
+- [ ] _Link to the operator PR (getting_started / kuttl)_
+- [ ] _Link to any other operator PRs (getting_started / kuttl)_
+- [ ] _Link to demo PR (raise against the `next` branch)_
+```
+
+<!--
+Make this a regular list so it isn't easily editable from the rendered
+description?
+-->
+```[tasklist]
+### Acceptance
+- [ ] Can build image locally
+- [ ] Kuttl smoke tests passes locally
+- [ ] Release notes written in a comment below
+- [ ] Applicable `release-note` label added to this issue
+```
+
+<details>
+<summary>Testing instructions</summary>
+
+```shell
+# See the latest version at https://pypi.org/project/image-tools-stackabletech/
+pip install image-tools-stackabletech==0.0.12
+
+bake --product opa=x.y.z # where x.y.z is the new version added in this PR
+
+kind load docker-image docker.stackable.tech/stackable/opa:x.y.z-stackable0.0.0-dev
+
+# Change directory into the opa-operator repository and update the
+# product version in tests/test-definition.yaml
+./scripts/run-tests --test-suite smoke-latest
```

Comment:
Question: do we have a test with rego rules? Maybe that would be better if the rules-version has changed.

Reply:
I'm not familiar with the opa-operator tests. In the context of the `opa` version bump, what changes _rules-version_? Is it the bundle builder?

Reply:
Actually, having looked at the tests again I think the smoke test is fine, as there is a rudimentary rule that is called. I think I was getting mixed up with a Trino [change](https://github.com/stackabletech/trino-operator/pull/494/files) that used different rule sets.